operations/use-trusty.yml

```diff
@@ -0,0 +1,5 @@
+- type: replace
+  path: /resource_pools/name=vms/stemcell?
+  value:
+    url: https://bosh.io/d/stemcells/bosh-aws-xen-hvm-ubuntu-trusty-go_agent?v=3586.79
```

Comment:
I don't know how I feel about hardcoding the version number here? Could we leave off the query parameter and have it pull the latest stem cell??

Reply:
I believe that will work until that hash changes right?

Reply:
Well I think this will hardcode the version nimber for the stemcell to always be that version. I think having it pull the latest is better.

Reply:
Updated.